src/adafruit_circuitplayground/utils.py

```diff
@@ -20,8 +20,8 @@ def show(state, debug_mode=False):
         if debug_mode:
             debugger_communication_client.update_state(json.dumps(message))
         else:
-            print(json.dumps(message) + '\0', end='',
-                  file=sys.__stdout__, flush=True)
+            # print(json.dumps(message) + '\0', end='',
```

Comment:
Perhaps we can remove the commented code if we don't need it? If someone wants to insert this code again in the future, they can find it in the commit history.

Reply:
oops did not mean to comment that out